README.md

```diff
@@ -39,8 +51,14 @@ make
 Finally, build and run the example app:
 
 ```
-cd examples/hello
-pipenv install
-spin build
+cd examples/hello_world
+../../target/release/spin-python app -o app.wasm
 spin up
 ```
+
+**Note:* In the samples provided in the `examples` directory, the `spin build` command depends on the plugin `py2wasm` being installed. Therefore, to test the locally built `spin-python` binary, replace the build command in the `spin.toml` to invoke it using spin build.
```

Comment:
Again, without the context of the different names, it is not clear how the locally build `spin-python` program relates to the `py2wasm` plugin it's replacing in the build command.